src/Models/ApplicationFormModel.php

```diff
@@ -36,4 +36,17 @@ public function getHearAbout()
         $result = $query->fetchAll();
         return $result;
     }
+
+    /**
+     * Gets all the sages from the database.
+     *
+     * @return array $result is data from the stages fields.
+     */
+    public function getStages()
```

Comment:
is this method needed as theres the getStageTitles() method now

Reply:
Took it out completely as it wasn't needed :)